gulp/styles.js

```diff
@@ -35,21 +35,31 @@ export function lint() {
 
 export function styles() {
   const minifyAssets = process.env.EQ_MINIMIZE_ASSETS === 'True'
+  const doSourceMaps = process.env.EQ_DEV_MODE === 'True'
```

Comment:
We are trying to stop the use of the generic "Dev Mode" so maybe if we want this feature it should have its own switch?